nomad/integration-tests-new.nomad

```diff
@@ -158,6 +168,15 @@ job "integration-tests-new" {
 
         KAFKA_BOOTSTRAP_SERVERS = var.kafka_bootstrap_servers
 
+        ORGANIZATION_MANAGEMENT_BIND_ADDRESS   = "0.0.0.0:1004" # not used but required due to clap
```

Comment:
Can you explain this comment a bit? If it's not used, why is it required? If it's required, why is it not used?

Reply:
I think I was mistakenly under the impression the organization-management client used this configuration: https://github.com/grapl-security/grapl/blob/internal-587-analyzers/src/rust/organization-management/src/lib.rs#L7-L8

but it doesn't appear to. I think we can remove this configuration value.

Reply:
I'm assuming it's because integration tests are going to have their ports assigned to them internally, but our configs require a valid endpoint

Reply:
Oh, wait, actually it's used here: https://github.com/grapl-security/grapl/blob/internal-587-analyzers/src/rust/organization-management/tests/test_create_organization.rs#L7

So if that environment variable is missing the clap configuration will fail to parse. The tests apparently only use that configuration for its database credentials and address.